src/pages/cards/demo/demo.html

```diff
@@ -0,0 +1,34 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>Cards</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
```

Comment:
Can we add the class `outer-content` here? Something to differentiate better between cards and the white background?